lib/foreman/renderer/scope/macros/snippet_rendering.rb

```diff
@@ -9,29 +9,29 @@ def snippet_if_exists(name, options = {})
 
           # provide embedded snippets support as simple erb templates
           def snippets(file, options = {})
-            if ::Template.where(:name => file, :snippet => true).empty?
-              render :partial => "unattended/snippets/#{file}"
-            else
+            if @source.find_snippet(file)
               snippet(file.gsub(/^_/, ""), options)
+            else
+              render :partial => "unattended/snippets/#{file}"
```

Comment:
I wonder how many people use this :-)

Reply:
I would remove that macro completely, to be honest. I don't get what it's supposed to do.